plugin-server/tests/clickhouse/e2e.test.ts

```diff
@@ -159,7 +159,8 @@ describe('e2e', () => {
         })
     })
 
-    describe('e2e export historical events', () => {
+    // historical exports are currently disabled
+    describe.skip('e2e export historical events', () => {
```

Comment:
non-blocking just curious how it works.

It looks like only historical export uses `fetchEventsForInterval`. Can other plugins use it?

Reply:
No, that's not exposed as a standalone util for plugins